ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -776,8 +776,10 @@ generate_default_config: < true | false | default -> true >
 ```yaml
 hardware_counters:
   features:
-    - <feature_1>: < direction | in | out >
-    - <feature_1>: < direction | in | out >
+    - name: <feature_1>
+      direction: < direction | in | out >
+    - name: <feature_1>
+      direction: < direction | in | out >
```

Comment:
We should replace this block with:
```suggestion
      direction: < in | out >
    - name: <feature_1>
      direction: < in | out >
```

to indicate that the only options are `in` and `out`